src/lib.rs

```diff
@@ -938,7 +938,7 @@ impl fmt::Display for LMS {
     }
 }
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Copy)]
```

Comment:
Do we really want to derive copy? I'd prefer if we explicitly call `.clone()` when we really want it. The struct contains 4 64bit floats, so it's *probably* cheaper to usually pass it by reference, not by value, right?